pipelines/main/misc/juliasyntax.test.yml

```
@@ -0,0 +1,204 @@
+              - "i686" # 32-bit Julia
+            julia_version:
+              # Don't put "1" here. Always include at least major and minor version.
+              - "1.0"
+              - "1.6" # old LTS
+              - "1.10" # current LTS
+              - "1.12" # latest stable
+        plugins:
+          - JuliaCI/external-buildkite#v1:
+              version: "./.buildkite-external-version"
+              repo_url: "https://github.com/JuliaCI/julia-buildkite"
+          - JuliaCI/julia#v1:
+              # This is the Julia install *outside* the sandbox.
+              # We only use this Julia install to run Sandbox.jl
+              # This is not the Julia install that will be used to run Pkg.test()
+              isolated_depot: true
+              # Drop default "registries" directory, so it is not persisted from execution to execution
+              persist_depot_dirs: packages,artifacts,compiled
+              version: '1.10' # current LTS. It doesn't really matter what version we pick here.
+              # arch: '{{matrix.julia_arch}}' # no need to specify the julia_arch for this "outside" Julia.
+          - staticfloat/sandbox#v2:
+              rootfs_url: https://github.com/JuliaCI/rootfs-images/releases/download/v8.0/agent_linux.i686.tar.gz
+              rootfs_treehash: "cba648ddb9ac0d292f9c8809652523806674a3d2"
+              uid: 1000
+              gid: 1000
+              workspaces:
+                # Include `/cache/repos` so that our `git` version introspection works.
+                - "/cache/repos:/cache/repos"
+          - JuliaCI/julia#v1:
+              # This is the Julia install *inside* the sandbox.
+              # That's the Julia install we actually use for Pkg.test()
+              isolated_depot: true
+              # Drop default "registries" directory, so it is not persisted from execution to execution
+              persist_depot_dirs: packages,artifacts,compiled
+              version: '{{matrix.julia_version}}'
+              arch: '{{matrix.julia_arch}}'
+        commands: |
+          echo "--- Julia versioninfo"
+          julia -e 'import InteractiveUtils; InteractiveUtils.versioninfo()'
+
+          echo "--- Pkg.dev-ing (by path) the local copy of JuliaSyntax"
+          export JULIA_PKG_PRECOMPILE_AUTO=0
+          # Double dollar signs $$ needed here because of Buildkite stuff
+          echo "Old JULIA_DEPOT_PATH: $${JULIA_DEPOT_PATH:?}"
+          # On Windows, the separator is ;
+          # On other platforms, the separator is :
+          sep=$$(julia -e 'println(Sys.iswindows() ? ";" : ":")')
+          # We change the depot path from `depot` to `depot:`
+          # The trailing colon/semicolon makes sure we can re-use the existing stdlib pkgimages,
+          # instead of needing to re-precompile them
+          # https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH
+          export JULIA_DEPOT_PATH="$${JULIA_DEPOT_PATH:?}$$(sep)"
```

@topolarity Can you apply this fix:

```suggestion
          export JULIA_DEPOT_PATH="$${JULIA_DEPOT_PATH:?}$${sep}"
```